content/docs/versions/1-7/fundamentals/key-concepts.md

```diff
@@ -72,3 +74,12 @@ _Events_ are either stored as a single _event_ or as an _event_ batch.
 
 The saved _events_ are later extracted from the Cassandra and displayed in the th2 test report in a chronologically and hierarchically organized manner.
 
+## Dictionary
+A _dictionary_ is static data/document shared between components.
+It is present in an infra-schema as a custom resource of kind `Th2Dictionary`.
+When an infra-schema is rolled out, the _dictionary_ is converted to a Kubernetes config map (base 64 encoded). 
+
+A common use case of a _dictionary_ is defining the protocol specification used by the customer.
+Messages created by the user are cross-referenced against the _dictionary_ to verify that the required information and message structure is present.
+This ensures that the messages created by th2 are standardized and accepted by the customer’s gateway.
+Each _dictionary_ is written for a single protocol only.
```

Comment:
I think information about restrictions is redundant, if you are not going into details

```suggestion
```